v3.0/reference/calicoctl/resources/node.md

```diff
@@ -43,6 +43,14 @@ spec:
 | Field       | Description                 | Accepted Values   | Schema | Default    |
 |-------------|-----------------------------|-------------------|--------|------------|
 | bgp      | BGP configuration for this node.  Omit if using Calico for policy only. | | [BGP](#bgp) |
+| orchRefs | Correlates this node to a node in another orchestrator. | | list of [OrchRefs](#OrchRef) |
+
+#### OrchRef
+
+| Field       | Description                 | Accepted Values   | Schema | Default    |
```

Comment:
I'd drop the word "other"  (a single NodeRef refers to a _specific_ orchestrator. 

Otherwise LGTM